src/DotNetStatus/Common/AnalyzerInfoSortFieldEnum.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DotNetStatus.Common
+{
+    public enum AnalyzerInfoSortFieldEnum
```

Comment:
All of these enums are not really a "Common" thing - they are for sorting. Also - could they be included within the controller (only if that's the only place that's using them)?